azkaban-web-server/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -341,6 +341,22 @@ private void handleExecutionJobDetailsPage(final HttpServletRequest req,
     page.add("jobLinkUrl", jobLinkUrl);
     page.add("jobType", node.getType());
 
+    final Status attemptStatus;
+    if (attempt == node.getAttempt()) {
+      attemptStatus = node.getStatus();
+    } else {
+      attemptStatus = node.getPastAttemptList().get(attempt).getStatus();
+    }
+    page.add("attemptStatus", attemptStatus.toString());
```

Comment:
ternary operator to make the code simpler?

Reply:
> ternary operator to make the code simpler?

@burgerkingeater thanks, done.